pep_sphinx_extensions/pep_theme/templates/partials/icons.html

```diff
@@ -0,0 +1,34 @@
+<!-- Adapted from Just the Docs → Furo -->
```

Comment:
Should this be `.html` or `.svg`? (I haven't used `<symbol />` before, so I don't know if it would make sense for this to be a "standalone" svg file.

Reply:
I think it doesn’t make that much sense to use it standalone, especially with that `display: none`.

But depending how we resolve https://github.com/python/peps/pull/2552#discussion_r857487860, we might not have to discuss this.